docs/resources/client/level-components/inner-level-string.md

```diff
@@ -49,6 +49,17 @@ The following keys were valid prior to 2.0 and are deprecated, since they are in
 | kS36 | Color 4      | 4                |
 | kS37 | 3DL Color    | 1003             |
 
+***Pre-1.9 Keys***
+
+The following keys were valid prior to 1.9 and were deprecated as of 1.9, since they were included in the keys ks29-33. Each color, instead of using one key, used 3 keys to represent Red, Green, and Blue values.
```

Comment:
When referring to keys and such, consider using the format `kS29`. Also, care for the capitalization.

In the Red, Green, Blue values, it's important to note that the aforementioned properties were respective to the keys, so change it to "... and Blue values, respectively."